consensus/src/lib.rs

```diff
@@ -99,40 +99,55 @@ impl<'a> ConsensusRef<'a> {
         let block_index = self.db_tx.get_block_index(block_id).map_err(BlockError::from)?;
```

Comment:
Isn't it more appropriate to call this function `get_parent`?

Reply:
@royster57 what do you think about `get_previous_block_index`?

Reply:
Sounds good to me.